suites/pacific/rados/tier-4_rados_tests.yaml

```diff
@@ -0,0 +1,124 @@
+# Suite contains basic tier-4 rados tests
+# conf - conf/pacific/rados/7-node-cluster.yaml
```

Comment:
Suggestion : even though the functional testing of this TC does not require big setup, I suggest we go with the 11 node cluster as new tests that would be added to this suite might need a larger setup.

Let me know what you think.

Reply:
Thanks Pawan, I have made the changes, switched to 11-node config